src/hotspot/share/interpreter/templateInterpreterGenerator.cpp

```diff
@@ -223,7 +223,9 @@ void TemplateInterpreterGenerator::generate_all() {
   method_entry(java_lang_Double_longBitsToDouble);
   method_entry(java_lang_Double_doubleToRawLongBits);
 
+#if 0
   method_entry(java_lang_continuation_doYield)
+#endif
```

Comment:
Leftover?

Reply:
Yes, I missed those.  Also, tests are failing because of missing push_cont_fastpath() call.  I think the right place to add it is in the c2i adapter.  I will remove from DRAFT mode when it's ready.